var/spack/repos/builtin/packages/r-crul/package.py

```diff
@@ -9,10 +9,11 @@
 class RCrul(RPackage):
     """crul: HTTP Client"""
 
-    homepage = "https://cran.r-project.org/package=crul"
-    url      = "https://cran.r-project.org/src/contrib/crul_0.7.4.tar.gz"
-    list_url = "https://cran.r-project.org/src/contrib/Archive/crul"
+    homepage = "https://cloud.r-project.org/package=crul"
+    url      = "https://cloud.r-project.org/src/contrib/crul_0.7.4.tar.gz"
+    list_url = "https://cloud.r-project.org/src/contrib/Archive/crul"
 
+    version('0.8.4', sha256='dbd950ad3b68402e5a5955615b1abcb5c9bdc846c93aa25f96a7a58913d04c8b')
     version('0.7.4', sha256='c963dd666ae3fc89b661ce19fce2fa19a16fc3825e1502105cae98ceb92c6014')
 
     depends_on('r-curl@3.3:', type=('build', 'run'))
```

Comment:
Missing dependency on `r-jsonlite`

Reply:
Ah, jsonlite was in the "suggests" list until version 0.8.4, which was added to CRAN on 08/02/2019. I had checked dependencies against version 0.7.4 but then I did another update round after that, and it moved from "suggests" to "imports" with that latest version. I added the dependency for r-jsonlite, with the version constraint.